pkg/state/state.go

```diff
@@ -765,6 +774,22 @@ func ReleaseToID(r *ReleaseSpec) string {
 	return id
 }
 
+func (st *HelmState) DeleteWait(args []string, release *ReleaseSpec) []string {
```

Comment:
could you add some unittests for this func?

Reply:
I added to cases into pkg/state/state_test.go